aio/content/examples/attribute-binding/src/app/app.component.css

```
@@ -1,22 +1,32 @@
 .special {
-  background-color: #1976d2;
+  background-color: black;
```

We can probably use hex codes for consistency:
```suggestion
  background-color: #000000;
```

---

OOC, why this suggestion?
(Personally I don't mind as long as we are consistent, but @jelbourn [prefers using `black` instead of the HEX value][1] :wink:)

[1]: https://github.com/angular/angular/pull/40622#discussion_r567987909

---

@gkalpak it just looks more consistent to use HEX, but I'm ok with other options if there is a strong preference :)

---

My preference is to prefer the named color values when they are available as they are more semantically meaningful.